Source/Android/arcana/threading/task_schedulers.h

```diff
@@ -27,6 +27,7 @@ namespace arcana
         } threadpool_scheduler{};
     }
 
+    template<size_t WorkSize>
```

Comment:
Elsewhere we default this template param to ``stdext::InplaceFunctionDefaultCapacity`. Should we do that here too?

Reply:
I was thinking the same thing, but not sure if we should. The only place that uses that is the inplace_function class itself. Dispatcher, for example, doesn't set a default.

Reply:
Oh interesting, I didn't remember that. Ok then I guess what you have is consistent.